ketcher-autotests/tests/Macromolecule-editor/Polymer-Bond-Tool/snake-bond-tool.spec.ts

```diff
@@ -0,0 +1,208 @@
+import { test } from '@playwright/test';
+import {
+  selectSingleBondTool,
+  selectSnakeBondTool,
+  waitForPageInit,
+} from '@utils';
+import { turnOnMacromoleculesEditor } from '@utils/macromolecules';
+/* eslint-disable no-magic-numbers */
+
+test.describe('Snake Bond Tool', () => {
+  test.beforeEach(async ({ page }) => {
+    await waitForPageInit(page);
+  });
+
+  test('Create snake bond between peptides', async ({ page }) => {
+    /* 
+    Test case: #3280 - Create snake bond 
+    Description: Snake bond tool
+    */
+
+    await turnOnMacromoleculesEditor(page);
+    await selectSnakeBondTool(page);
+
+    await page.getByText('Tza').click();
+
+    await page.mouse.click(300, 300);
+    await page.mouse.click(400, 400);
+    await page.mouse.click(300, 500);
+    await page.mouse.click(200, 200);
+
+    const peptides = await page.getByText('Tza').locator('..');
+    const peptide1 = peptides.nth(0);
+    const peptide2 = peptides.nth(1);
+    const peptide3 = peptides.nth(2);
+    const peptide4 = peptides.nth(3);
+
+    await selectSingleBondTool(page);
+
+    await peptide1.hover();
+    await page.mouse.down();
+    await peptide2.hover();
+    await page.mouse.up();
+
+    await page.mouse.down();
+    await peptide3.hover();
+    await page.mouse.up();
+
+    await page.mouse.down();
+    await peptide4.hover();
+    await page.mouse.up();
+
+    await page.screenshot({
+      path: 'tests/Macromolecule-editor/screenshots/create-snake-bond-between-peptides.png',
+      fullPage: true,
+    });
```

Comment:
I didn`t use takeEditorScreenshot here because I wanted to have fullPage screenshot for showing chosen mode btn. Should I replace it to takeEditorScreenshot in this case?

Reply:
I noticed there is a takePageScreenshot func for taking the whole page screenshot